lib/v2/api.js

```diff
@@ -88,7 +88,7 @@
          * @param {Function} [cb(data)] Callback
          */
         apply: function(element, cb) {
-            var teamslug = element.getAttribute('data-gratipay-teamslug');
+            var projectslug = element.getAttribute('data-gratipay-teamslug');
```

Comment:
We can't use `data-gratipay-team-slug` (with the dash) in the widget itself (that the user sets up) because that will break existing widgets. Or maybe I'm misunderstanding you?

E.g., `var projectid = element.getAttribute('data-gratipay-teamslug');` will work, and `var projectid = element.getAttribute('data-gratipay-team-slug');` will break.

(Also can't use hyphens in javascript variables, if that's what you mean.)